### PR TITLE
Add missing "CURRENTFILE" variable content

### PR DESCRIPTION
**Pull request by whongx**
Just found the CURRENTFILE variable content is empty.

### CODE REVIEW

**Comment by ayufan** [comment]
Plese remove CURRENTFILE completely as it is not used in that case.

**Comment by whongx** [comment]
ok!

File: package/root/usr/local/sbin/pine64_update_package.sh [hunk 1]
```diff
@@ -50,6 +44,4 @@ if [ -z "$MARK_ONLY" ]; then
 	echo "Done - you should reboot now."
 else
 	echo "Mark only."
-fi
-
-echo $ETAG > "$CURRENTFILE"
+ficu
```
Comment (ayufan) [inline package/root/usr/local/sbin/pine64_update_package.sh]: Do we need the `ficu`?

Comment (whongx) [inline package/root/usr/local/sbin/pine64_update_package.sh]: What is ficu?

Comment (ayufan) [inline package/root/usr/local/sbin/pine64_update_package.sh]: It is added by your commit.

Comment (whongx) [inline package/root/usr/local/sbin/pine64_update_package.sh]: Very sorry.. typo!

**Pull request merged**